src/Sarif/Autogenerated/Location.cs

```diff
@@ -40,7 +41,12 @@ public virtual SarifNodeKind SarifNodeKind
         [DataMember(Name = "id", IsRequired = false, EmitDefaultValue = false)]
         [DefaultValue(-1)]
         [JsonProperty(DefaultValueHandling = DefaultValueHandling.IgnoreAndPopulate)]
-        public virtual int Id { get; set; }
+        public virtual BigInteger Id { get; set; }
+
+        public bool ShouldSerializeId()
+        {
+            return Id != -1;
+        }
```

Comment:
this is needed because DefaultValue does not work for BigInteger.

then this need 2 things,
1. in the constructor to set it, which we already have.
2. ShouldSerializeXXX to handle the serialization.